go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/google/martian v2.1.1-0.20190517191504-25dcb96d9e51+incompatible // indirect
 	github.com/google/uuid v1.2.0
 	github.com/gophercloud/gophercloud v0.17.0
-	github.com/gophercloud/utils v0.0.0-20210323225332-7b186010c04f
+	github.com/gophercloud/utils v0.0.0-20210530213738-7c693d7efe47
```

Comment:
I think the go.mod changes should be in the update-vendor commit?

Reply:
Yep. Sorry got my commits mixed up. Will fix it in the next push